src/scripts/main.js

```diff
@@ -1,3 +1,28 @@
 'use strict';
 
 // write your code here
+const h1 = document.querySelector('.logo');
+const body = document.body;
+
+const promise2 = () =>
+  new Promise((resolve, reject) => {
+    setTimeout(() => {
+      const div = document.createElement('div');
+
+      div.classList.add('message', 'error-message');
+      div.textContent = `Promise was rejected!`;
+      body.append(div);
+    }, 3000);
+  });
```

Comment:
The `promise2` function is missing a call to `reject()`. According to the task description, `promise2` should be rejected after 3 seconds. You need to call `reject()` inside the `setTimeout` function.